front/tests/012-op-simulation-settings-tab.spec.ts

```diff
@@ -0,0 +1,356 @@
+import { test } from '@playwright/test';
+import { v4 as uuidv4 } from 'uuid';
+
+import type {
+  ElectricalProfileSet,
+  ElectricalProfileSetData,
+  Project,
+  Scenario,
+  Study,
+} from 'common/api/osrdEditoastApi';
+
+import HomePage from './pages/home-page-model';
+import OperationalStudiesInputTablePage from './pages/op-input-table-page-model';
+import OperationalStudiesOutputTablePage from './pages/op-output-table-page-model';
+import OperationalStudiesSimulationSettingsPage from './pages/op-simulation-settings-page-model';
+import OperationalStudiesTimetablePage from './pages/op-timetable-page-model';
+import OperationalStudiesPage from './pages/operational-studies-page-model';
+import ScenarioPage from './pages/scenario-page-model';
+import { readJsonFile } from './utils';
+import { deleteApiRequest, postApiRequest } from './utils/api-setup';
+import { cleanWhitespace, type StationData } from './utils/dataNormalizer';
+import setupScenario from './utils/scenario';
+import scrollContainer from './utils/scrollHelper';
+import enTranslations from '../public/locales/en/timesStops.json';
+import frTranslations from '../public/locales/fr/timesStops.json';
+
+const freakRollingStockName = 'freak_rolling_stock_test_e2e';
```

Comment:
I'm really not enthusiast about this name, why "freak"? couldn't we choose a better and more explicit name?

Reply:
The used RS contains exaggerated values and noticeable discrepancies across the various charts.
I’m open to any other suggestions